pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
 
-- `ca_certificate_path`
-- `ca_certificate_key_path`
-- `ca_certificate_chain_path`
+# What is supported and what is not
+- Only custom CA certificates that have an associated RSA-type key pair are supported. Other types of certificates, such as those using ECC-type key pair, are currently not supported. Support for ECC-type key pairs will be added in the future.
+- You have to use a custom CA certificate only with a fresh installation of DC/OS Enterprise 1.10 or higher. Older versions of DC/OS are not supported.
 
-**Note:** 
+# Glossary of the terms used in this documentation
+- Custom CA certificate
 
-- Only custom CA certificates that have an associated RSA-type key pair are supported.
-- You can use a custom CA certificate only with a new 1.10 install.
+Your custom CA certificate in the PEM format, which will be used to issue certificates for DC/OS components such as Admin Router. The custom CA certificate is either an intermediate CA certificate (issued by another CA) or a root CA certificate (self-signed by the custom CA).
 
-# Procedure
+- Private key associated with the custom CA certificate
 
-1. Using a [custom setup file and the CLI or advanced installer](/1.10/installing), provide the paths to the custom CA certificate, corresponding private key, and optionally corresponding intermediate CA certificates and/or a root CA certificate as part of the chain. All files must be placed in `genconf` directory in the home directory of your bootstrap node. The installer verifies the data.
-1. Before installing DC/OS to all nodes, place the private key file corresponding to the custom CA certificate specified in `ca_certificate_key_path` securely on all master nodes in `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The file must be readable by the root user and have 0600 permissions set.
+The private key in the PKCS#8 format associated with the custom CA certificate.
+
+- Certificate chain associated with the custom CA certificate
+
+The complete CA certification chain required for end-entity certificate verification. It has to include the certificates of all parent CAs of the intermediate custom CA up to and including the self-signed root CA certificate. Empty, if the custom CA certificate is a self-signed root CA certificate.
+
+- Configuration
+
+The set of the configuration parameters that governs the specific aspects of the installation procedure. The configuration is stored in the configuration file.
+
+- DC/OS Configuration file
+
+The file which contains the DC/OS configuration parameters, in particular, the  filesystem relative paths to the custom CA certificate, associated private key and the certificate chain. 
```

Comment:
Let's either leave out ", in particular, the  filesystem relative paths to the custom CA certificate, associated private key and the certificate chain." or improve the grammar of this sentence.